text_generation/causal_lm/cpp/continuous_batching/library/src/scheduler.hpp

```diff
@@ -55,6 +55,8 @@ class Scheduler {
             }
         }
 
```

Comment:
we can move setting out of memory status here as well from `step()` method

Reply:
Yes, done.